packages/core/utils/src/common/to-handle.ts

```diff
@@ -0,0 +1,18 @@
+import { kebabCase } from "./to-kebab-case"
+
+/**
+ * Helper method to create a to be URL friendly "handle" from
+ * a string value.
+ *
+ * - Works by converting the value to lowercase
+ * - Splits and remove accents from characters
+ * - Removes all unallowed characters like a '"%$ and so on.
+ */
+export const toHandle = (value: string): string => {
```

Comment:
Suggestion : to handle looks a bit tight to a product specific knowledge. It kind of leaks an internal detail from a domain. Would it make sense to name it differently? I dont have a name in my head straight away though 🥹

Reply:
Lets keep it as is for now and we can circle back on that naming later